ledger-core/v2/virtual/integration/method_test.go

```diff
@@ -323,11 +323,11 @@ func TestVirtual_Method_WithExecutor(t *testing.T) {
 			callResultPl := metaPayload.(*payload.Meta).Payload
 			callResultPlType, err := payload.UnmarshalType(callResultPl)
 			assert.NoError(t, err)
-			assert.Equal(t, payload.TypeVCallResult, callResultPlType)
+			assert.Equal(t, payload.TypeVStateRequest, callResultPlType)
```

Comment:
I've wrote a bad test, we need to prepare an object, before (so no VStateRequest is needed)
Later we should obtain VCallResult

Reply:
`Method_PrepareObject` doesn't prepare an object?

Reply:
Or I can use it?

Reply:
done